.github/workflows/appimage.yml

```diff
@@ -0,0 +1,57 @@
+name: Build AppImage rplidar module
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ 'main' ]
+    paths-ignore:
+      - 'README.md'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
I think you can remove the tags here, as we don't need to do tagging for this repo.